examples/tv-casting-app/tv-casting-common/support/ChipDeviceEventHandler.cpp

```diff
@@ -105,17 +105,19 @@ void ChipDeviceEventHandler::HandleFailSafeTimerExpired()
     }
 
     // if UDC was NOT in progress (when the Fail-Safe timer expired), start UDC
-    ChipLogProgress(AppServer, "ChipDeviceEventHandler::HandleFailSafeTimerExpired when sUdcInProgress: %d, starting UDC",
+    ChipLogProgress(AppServer, "ChipDeviceEventHandler::HandleFailSafeTimerExpired() when sUdcInProgress: %d, starting UDC",
```

Comment:
It doesn't look like the new commissioning flow is implemented in the case of this HandleFailSafeTimerExpired() callback?

Reply:
I realized that no changes we're necessary here since we need to call OpenBasicCommissioningWindow() for both Commissionee-Generated passcode (AccountLogin) and Commissioner-Generated passcode flows.